### PR TITLE
Vagrant stops installing pulp-server rpm

### PR DESCRIPTION
**Pull request by werwty**
This fix requires https://github.com/pulp/pulp/pull/2959 to be merged first

closes #2497
https://pulp.plan.io/issues/2497

### CODE REVIEW

**Comment by werwty** [comment]
With this fix
`[vagrant@dev ~]$ sudo dnf list installed | grep pulp`
`[vagrant@dev ~]$`

**Comment by bmbouter** [comment]
I tested this. I vagrant upped with this PR checked out and pulp/pulp master at 18f7c6ae1aa8c8a430e3fd2260ab11f8bdfd9078 which includes pulp/pulp#2959. Vagrant finished as expected and indeed I get the following output:

```
[vagrant@dev ~]$ sudo dnf list installed | grep pulp
python-kombu.noarch                    1:3.0.33-6.pulp.fc24            @pulp-nightlies
python-nectar.noarch                   1.5.3-1.fc24                    @pulp-nightlies
```

File: ansible/roles/lazy/tasks/main.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@
   with_items:
       - squid
       - httpd
-      - python-pulp-streamer
```
Comment (bmbouter) [inline ansible/roles/lazy/tasks/main.yml]: Don't we want this to run inside of vagrant though?

Comment (werwty) [inline ansible/roles/lazy/tasks/main.yml]: We don't need to install python-pulp-streamer via dnf when it's being installed with setuptools
https://github.com/pulp/pulp/blob/master/streamer/setup.py

This is also what was pulling in the unwanted pulp-server dependency before.

**Pull request merged**